docs/excel/excel-add-ins-tables.md

```diff
@@ -233,6 +232,30 @@ Excel.run(function (context) {
 
 ![Table data in Excel](../images/excel-tables-get-data.png)
 
+### Detect data changes
+
+Your add-in may need to react to users changing the data in a table. To detect these changes, [Tables](/javascript/api/excel/excel.table) provide an `onChanged` event. This event fires whenever the data is changed in a particular table. Event handlers for `onChanged` receive a [TableChangedEventArgs](/javascript/api/excel/excel.tablechangedeventargs).
```

Comment:
Suggest that you leave the first sentence as-is and then replace the other 3 sentences with the following:

To detect these changes, you can [register an event handler](excel-add-ins-events.md#register-an-event-handler) for the `onChanged` event of a [table](/javascript/api/excel/excel.table). Event handlers for the `onChanged` event receive a [TableChangedEventArgs](/javascript/api/excel/excel.tablechangedeventargs) object when the event fires.

(You can edit this comment in GitHub to copy/paste this markdown if you'd like to use it.)